File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioFieldSetOfRole.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnThemePickMultiVariant;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdRole;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldSetOfRole extends StudioFieldEditable
{
  @Nullable
  public MetaIdField defaultRoleFieldId;

  @Nullable
  public MetaIdRole[] defaultRoleIdSet;

  @Nullable
  public EnumDefnThemePickMultiVariant showAs;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldSetOfRole.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnThemePickMultiVariant
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdRole
import com.neome.api.meta.base.dto.StudioFieldEditable

open class StudioFieldSetOfRole : StudioFieldEditable()
{
  var defaultRoleFieldId: MetaIdField? = null
  var defaultRoleIdSet: Array<MetaIdRole>? = null
  var showAs: EnumDefnThemePickMultiVariant? = null
}